<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MantidSampleMaterialDialog</class>
 <widget class="QDialog" name="MantidSampleMaterialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>302</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sample Material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeMaterialProperties">
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Property</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbSetMaterial">
       <property name="text">
        <string>Set Material</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCopyMaterial">
       <property name="text">
        <string>Copy Material</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
